catalog/ui/catalog-ui-search/src/main/java/org/codice/ddf/catalog/ui/security/AccessControlPolicyExtension.java

```diff
@@ -76,8 +82,15 @@ public AccessControlPolicyExtension(
 
     hasAccessGroups = predicate(Constants.ROLES_CLAIM_URI, SecurityAttributes.ACCESS_GROUPS);
 
+    hasAccessGroupsReadOnly =
+        predicate(subjectIdentity.getIdentityAttribute(), SecurityAttributes.ACCESS_GROUPS_READ);
```

Comment:
Should this be `Constants.ROLES_CLAIM_URI`?

Reply:
yes, it should be.